src/workers/postgres.rs

```diff
@@ -115,6 +121,21 @@ impl PostgresSession {
             )
             .await?;
 
+        let insert_5_batch_block_statement = client
```

Comment:
Why limit to 5, you could make it configurable. May be variadics can help with that.